### PR TITLE
DOCS: Docker-compose commands updated

### PR DESCRIPTION
**Pull request by waldbauer-certat**
Updated docker-compose install instructions

### CODE REVIEW

**Comment by wagner-certat** [comment]
A note to install docker _and docker-compose_ can also be added (i.e. the programs are required and not installed automatically as dependency or whatsoever)

File: docs/user/FAQ.rst [hunk 1]
```diff
@@ -131,3 +131,29 @@ Multithreading is not available for some bots and AMQP broker is necessary. Poss
...
 If you think this mapping is wrong, please report a bug.
+
+.. _docker security headers:
+Docker: Security Headers
+-------------------------------------------------------------------
+
+If you run our docker image in production, we recommend you to set security headers.
+You can do this by creating a new file called ``example_config/nginx/security.conf`` in the cloned ``intelmq-docker`` repository.
+
+Write the following inside the configuration file, and edit the ``http(s)://<your-domain>`` to your domain name.
```
Comment (wagner-certat) [inline docs/user/FAQ.rst:142]:
```suggestion
Write the following inside the configuration file, and change the ``http(s)://<your-domain>`` to your domain name.
```

**Review by wagner-certat — CHANGES_REQUESTED** [review]

**Review by wagner-certat — APPROVED** [review]

File: docs/user/intelmq-api.rst [hunk 2]
```diff
@@ -199,7 +199,7 @@ Can I just install it from the deb/rpm packages while installing IntelMQ from a
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Yes, you can install the API and the Manager from the deb/rpm repositories, and install your IntelMQ from a somewhere else, e.g. a local repository.
-However, knowledge about Python system administration experience and is recommended if you do so.
+However, knowledge about Python system administration experience is recommended if you do so.
```
Comment (schacht-certat) [inline docs/user/intelmq-api.rst:202]: That `and` was probably at the wrong place and it should be `Python **and** system administration experience`

**Comment by wagner-certat** [comment]
@waldbauer-certat This can be merged

**Comment by wagner-certat** [comment]
Cherry-picked for maintenance

**Pull request merged**